base/doc/ltnews42.tex

```diff
@@ -201,10 +201,51 @@ \subsection{Requiring or testing for the Tagging support code}
 the format. To test whether the PDF management has been loaded, the test
 \cs{IfPDFManagementActiveTF} is provided.
 
+ 
+\subsection{Moving paragraph tagging into sockets}
+
+Paragraphs in \LaTeX{} can be nested, e.g., you can have a paragraph
+containing a display quote, which in turn consists of more than one
+(sub)paragraph, followed by some more text which all belongs to the
+same outer paragraph.
+
+To model such \enquote{big paragraphs} \LaTeX{} uses a structure named 
```

Comment:
I like the name "semantic paragraph" that you used elsewhere. That might be what we have been looking for and would also, allow to use para/semantic  (instead of para/main) which I think is better than using "main"